### DISCUSSION TITLE
Can we Install WARPX on windows platform?

### DISCUSSION THREAD

**Discussion by Vyaghri-Sista** [BLAST-WarpX/warpx#5204] [Installation] [answered]
Hi, I want to do some studies of Space charge compensation studies in our LEBT line consists of two solenoids. I came to know that these type of studies can be done using WARPX. Can I install this WARPX on my windows laptop?

**Comment by RemiLehe** [comment, marked as answer]
Thanks for the question.
Yes, absolutely: WarpX can be installed on Windows. We recommend using `conda` for this, as explained here:
https://warpx.readthedocs.io/en/latest/install/users.html
